### DISCUSSION TITLE
C++Math.cc error

### DISCUSSION THREAD

**Discussion by netskyinc** [LeoNatan/LNPopupController#476] [General] [answered]
Greetings. Trying to run example.

error: Build input file cannot be found: '/Users/netskyinc/Desktop/LNPopupController/LNPopupController/LNPopupController/Private/C++Math.cc' (in target 'LNPopupController' from project 'LNPopupController')

**Comment by LeoNatan** [comment, marked as answer]
This was reported here: https://github.com/LeoNatan/LNPopupController/issues/474

It has been fixed in the latest version.
